src/user/userData.ts

```diff
@@ -1,8 +1,10 @@
 import { request, gql, GraphQLClient } from 'graphql-request'
-import { userQuery } from '../constants/graphqlQueries';
+import { generateL2RasterProductQuery, userProductsQuery, userQuery } from '../constants/graphqlQueries';
+import { CurrentUser, Product, UserResponse } from '../types/graphqlTypes';
+import { GeneratedProduct, getUserProductsResponse } from '../types/constantTypes';
 
 const userIdQuery = gql`${userQuery}`
-const baseUri = 'https://d15gds5czd9p7k.cloudfront.net';
+const baseUri = process.env.REACT_APP_SWODLR_API_BASE_URI;
```

Comment:
Is it really required that we prefix all the environment variables with `REACT_APP_`? Could we just prefix it with `SWODLR_`?

Reply:
REACT_APP_ is required for all react env variables